src/content/concepts/configuration.md

```diff
@@ -46,8 +47,12 @@ module.exports = {
 };
 ```
 
+_See_: [Configuration section](/configuration) for the all supported configuration options
```

Comment:
can you please change to `/configuration/` so it also works on dev ? dev server doesn't work for non trailing slash urls